### ISSUE TITLE
docs: enhancing "using docker" section

### ISSUE THREAD

**Issue by danielhe4rt** [scylladb/scylla-monitoring#2294]
Hey folks, how y'all doing? I would like to ask for an enhancement in the next versions of the [docker-compose](https://monitoring.docs.scylladb.com/branch-4.7/install/docker-compose) page. 

### Motivation
Recently I delve into Scylla Monitoring and I still think that is missing some information related to "versioning" in commands, more variety of commands or even updated versions. 

![image](https://github.com/scylladb/scylla-monitoring/assets/6912596/74e235c3-97d3-4c04-8986-ec4e2d13abe7)

When I saw the "-v 2020.1" I thought "ok, maybe we stopped in this version." but seems to be other ones. And where can I find this other versions from Open Source and Enterprise?

###  Suggest a fix

I'd like to keep track on the latest stable versions in the docs or even using Sphinx Variables to populate this for us like a flag. Something like:

-----
#### Grafana Dashboard Load file

To set the dashboard load file, you can run the ./generate-dashboards.sh with the -t command line flag and the -v flag to specify the version:

```bash
# Open Source Dashboard
./generate-dashboards.sh -t -v 6.0

# Enterprise Dashboard
./generate-dashboards.sh -t -v 2024.1
```

This command generates the files under: `grafana/provisioning/dashboards/` and you can find the supported versions [here](/);

----

**Comment by mykaul** [comment]
https://monitoring.docs.scylladb.com/branch-4.7/reference/matrix.html ?

**Comment by danielhe4rt** [comment]
Yeah! The goal is to make sure that we will have easy forwarding when someone needs a specific information about versioning.

**Comment by mykaul** [comment]
@DanielHe4rt - so I'm unsure - what's missing?

**Comment by amnonh** [comment]
@tzach we have a file with all the parameters, is there a way to pull those parameters from there and propagate them?

**Comment by tzach** [comment]
For sure there is a way, not sure its worth the effort...
Where is this file?

**Comment by amnonh** [comment]
it's called ```versions.sh``` in the root directory of the project

**Comment by tzach** [comment]
I'm not sure publishing https://github.com/scylladb/scylla-monitoring/blob/master/versions.sh will be useful.
Can you add description to each?
We can also link to this file, it is public.

**Comment by amnonh** [comment]
@tzach this is definitely not what I suggest, I would like the Python script to read the file and use that information inside the docs
